examples/Quasiconvex Optimization by Bisection.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solve the Quasiconvex problem by Convex feasibility and Bisection Search\n",
+    "A quasiconvex problem is defined as - \n",
+    "\n",
+    "\\begin{array}\n",
+    "   \\mbox{minimize}    & f_0(x)\\\\\n",
+    "   \\mbox{subject to}  & f_i(x) \\leq 0  \\\\ \n",
+    "   & Ax = b\n",
+    "\\end{array}\n",
+    "\n",
+    "where $f_0 : \\mathbb{R}^n \\rightarrow \\mathbb{R}$ is Quasiconvex and the remaining functions are convex. \n",
+    "\n",
+    "Here we can have locally optimal solutions that are not globally optimal.\n",
+    " \n",
+    "Quasiconvex functions have the useful property that their sub-level sets are convex. We can exploit this to construct an algorithm to solve a Quasiconvex Problem.\n",
+    "\n",
+    "If $f_0$ is Quasiconvex, then there exists a family of functions $\\phi_t(x)$ that are convex in $x$ for a fixed $t$ such that - $f_0(x) \\leq t \\Leftrightarrow \\phi_t(x) \\leq 0$\n",
```

Comment:
This is very confusing and not very helpful: how do we find phi? Much better is to make a concrete, constructive statement: if f_0 is quasiconvex, then the t-th sublevel set {x: f_0(x) \leq t} is convex, for any t. So finding a point in the t-th sublevel set is a convex optimization problem.